test/pricing-poller_test.js

```diff
@@ -130,7 +131,42 @@ describe('Pricing', () => {
       }],
     });
 
-    //describeSPHStub.onSecondCall().throws();
+    describeSPHStub.onSecondCall().callsFake((ec2, method, params) => {
+      assume(params.NextToken).is.equal('abc123');
+      return {
+        SpotPriceHistory: [{
+          Timestamp: '2017-07-05T13:11:21.000Z',
```

Comment:
Can we bumb the timestamps and SpotPrices to ensure that there's a test case for a price that's from the first request and another from the second request?

Also, AvailabilityZone would probably be better in the full `us-east-1d` format, instead of `d`